TRANSLATION_WORKFLOW.md

```diff
@@ -0,0 +1,282 @@
+  - [You have some files that you want to add to the `translations` queue.](#you-have-some-files-that-you-want-to-add-to-the-translations-queue)
+  - [There is a file that hasn’t been translated before into that language](#there-is-a-file-that-hasnt-been-translated-before-into-that-language)
+  - [There is a file where you want to re-run a translation but hasn’t been edited recently](#there-is-a-file-where-you-want-to-re-run-a-translation-but-hasnt-been-edited-recently)
+- [Key concepts / Glossary](#key-concepts--glossary)
+  - [GitHub Branches and Pull Requests](#github-branches-and-pull-requests)
+  - [GitHub Actions](#github-actions)
+- [PostgreSQL tables](#postgresql-tables)
+  - [Translations](#translations)
+  - [Jobs](#jobs)
+  - [TranslationsJobs](#translationsjobs)
+  - [Locales](#locales)
+  - [Statuses](#statuses)
+  - [Serialization / Deserialization](#serialization--deserialization)
+- [Gatsby Cloud](#gatsby-cloud)
+- [Context](#context)
+<!-- toc -->
+
+# High-level overview
+
+## Purpose
+
+The site has content & accepts contributions in english, but users can be located anywhere in the world. The translation process takes the english content and translates it into other languages.
+
+**NOTE:** "TV" means "Translation Vendor" in the context of this document.
+
+## What is the translation process
+
+The translation process has the following high-level steps:
+
+1. When a PR to main is merged, if it includes translatable files, queue those files up to be translated.
+2. Submit queued translation requests to vendor.
+
+   - We have a batching function that will only allow us to fetch 50 file URIs at a time. This is due to the TV API design where file URIs are fetched via query parameters instead of part of the body of the API request.
+
+3. Check for finished translations, download translated files and add them to the repo.
+
+Additionally, we've recently added the ability to have content be machine translated, but the process is ultimately still the same, we just decide in the first step if content should be machine or human translated.
+
+**NOTE:** Check out the workflow Figma design for a visual reference [here](https://www.figma.com/file/Ph8QXJlXm2Xxu2As27HBqx/Send-Translation?node-id=0%3A1).
+
+# Detailed Overview
+
+## Background
+
+The whole process is executed as a series of GitHub workflows which can be found [here](https://github.com/newrelic/docs-website/tree/develop/.github/workflows).
+
+The workflows are:
+
+- queueing translation requests
+  - this workflow takes a PR, looks for translatable files, and queues them up for either human or machine translation
+- sending content for machine translation
+  - this workflow takes queued up machine translation requests, and submits them to the vendor
+- sending content for human translation
+  - this workflow takes queued up human translation requests, and submits them to the vendor
+- checking for finished translations, downloading content, cleaning up
```

Comment:
i would capitalize these workflow names to match how they are in the actions tab (and maybe bold/italicize) for clarity